package.json

```
@@ -32,8 +33,10 @@
     "unist-util-visit": "^2.0.0"
   },
   "devDependencies": {
+    "@types/hast": "^2.3.1",
```

unless there is a specific hast patch/minor version being relied on, it's good to use a wider semver range.
```suggestion
    "@types/hast": "^2.0.0",
```